lib/dialects/mysql/query-generator.js

```diff
@@ -388,7 +388,7 @@ class MySQLQueryGenerator extends AbstractQueryGenerator {
     if (!typeWithoutDefault.has(attributeString)
       && attribute.type._binary !== true
       && Utils.defaultValueSchemable(attribute.defaultValue)) {
-      template += ` DEFAULT ${this.escape(attribute.defaultValue)}`;
+      template += ` DEFAULT (${this.escape(attribute.defaultValue)})`;
```

Comment:
```js
If (attribute.defaultValue instanceof Utils.Fn) {
   // add () to DEFAULT
} else {
  // keep old code
}
```

Reply:
This should fix issue for both MySQL 5.7 & 8

Reply:
Thank You @sushantdhiman. I'll make some tests for this.